ovirtlib4/system_service.py

```diff
@@ -64,8 +70,96 @@ def _get_collection_entity(self):
         """
         return CollectionEntity(connection=self.connection)
 
-    def get(self, *args, **kwargs):
-        """ Same as list() """
+    def _process_sample(self, wait_for, wait_method, sample):
+        """
+        Handle a result sample according to the given 'wait_for' and 'wait_method'
+
+        Args:
+            wait_for (object): see run_sampler() docstring
+            wait_method (str): see run_sampler() docstring
+            sample (list): List of SDK entities, returned by get()
+
+        Returns:
+            list: If 'str' or 'bool' return the given sample if it match,
+                if 'str' return the filtered list if it match,
+                if no match return an empty list
+        """
+        if type(wait_for) == bool:
+            if (not sample) == (not wait_for):
```

Comment:
maybe it will be better:
if bool(sample) == wait_for:
or:
if (not sample) != wait_for:

to me it looks more clean and readable

Reply:
Yap, this if bool(sample) == wait_for better